File: case-study-video/generate-voiceover.mjs
import { ttsSave } from './node_modules/edge-tts/out/index.js'

const script = `
Meet La Ultra: Run and Bee, a movement and mindfulness app by Dr. Rajat Chauhan.
Before you train, it asks the question most fitness apps skip: how does your body feel today?
Start with a two-minute Feel check-in across body, mind, and movement.
Your score can adjust the day's run, so the plan bends before your body breaks.
Then breathe at five breaths per minute with a calm timer and metronome.
Open the movement library for functional tests, strength tools, and running drills, each with precise clinical cues.
When you're ready to run, the coach builds weekly plans, calculates pace zones, logs check-ins, and answers running questions with your recent history in view.
History turns Feel, workouts, readiness, and quality into trends you can understand.
Sign in with Google or use guest mode.
Add it to your home screen; it works offline.
La Ultra: Run and Bee.
Listen first. Then move.
`

await ttsSave(script.trim(), './public/commercial-voice-soft.mp3', {
  voice: 'en-US-JennyNeural',
  rate: '+2%',
  pitch: '-2Hz',
  volume: '+0%',
})

File: case-study-video/generate_voiceover.py
import asyncio
import sys
from pathlib import Path

sys.path.insert(0, str(Path(__file__).parent / "pydeps"))

import edge_tts

SCRIPT = """
La Ultra, Run and Bee is a portfolio case study about designing with one real person.
Dr. Rajat Chauhan is a sports medicine expert, author, runner, and my father.
His problem was clear: runners have more data than ever, but many have stopped listening to their bodies.

The research came from interviews, observation, and his clinical language.
The first prototypes failed in useful ways: streak pressure, generic cues, busy motion, and the wrong front door.

So the app changed.
It starts with Feel, a two minute readiness check.
It guides breathing with a calm orb.
It teaches movement through exact clinical cues.
And the running coach asks how the body feels before giving pace.

The evidence is the process: daily feedback, real testing, and one hundred sixty commits across sixteen active build days.
Not a fitness dashboard.
A practice for listening first, then moving.
"""


async def main():
    out = Path(__file__).parent / "public" / "case-study-voice-portfolio-58.mp3"
    communicate = edge_tts.Communicate(
        SCRIPT.strip(),
        voice="en-US-JennyNeural",
        rate="+8%",
        pitch="-2Hz",
        volume="+0%",
    )
    await communicate.save(str(out))
    print(out)


asyncio.run(main())
